ha_cluster_exporter.go

```diff
@@ -151,6 +151,11 @@ func main() {
 	// set DRBD metrics
 
 	go func() {
+        if _, err := os.Stat("/sbin/drbdsetup"); os.IsNotExist(err) {
+                log.Warnln("drbdsetup binary not available, DRBD metrics won't be collected")
+                return
+        }
+
```

Comment:
the indentation is off, could you fix it? Also in Go tabs are used instead of spaces!

Reply:
Yep, I use tabs as standard - not sure what happened there, but it's fixed! :+1: